### ISSUE TITLE
Clear activities while transaction is pending

### ISSUE THREAD

**Issue by MareenG** [gnosis/gp-swap-ui#228]
If I clear my recent activity history while a transaction is still pending, the pending icon in the top left corner disappears. 
This might give the impression that I canceled my order, but it is not the case.

<img width="1038" alt="Bildschirmfoto 2021-03-11 um 10 50 47" src="https://user-images.githubusercontent.com/34510341/110769380-677e5e80-8258-11eb-9525-b84dfd1ea1b6.png">
<img width="927" alt="Bildschirmfoto 2021-03-11 um 10 50 53" src="https://user-images.githubusercontent.com/34510341/110769384-69482200-8258-11eb-9fc9-d5a9a312fa7d.png">
<img width="989" alt="Bildschirmfoto 2021-03-11 um 10 51 17" src="https://user-images.githubusercontent.com/34510341/110769388-6a794f00-8258-11eb-85b2-13f0be9b053d.png">

**Comment by W3stside** [comment]
hmmmmmmmmmmm

good catch, though I'm not really sure that's a bug. Should the intended UX be that clearing the recent activities doesn't clear the pending spinner?

imo it should clear it, but i can also see the other case being valid too

**Comment by MareenG** [comment]
Yeah it is a UX thing.
I feel that the pending spinner shouldn't be removed when clearing the history as it could indicate that the transaction could be canceled. But this is just my experience. I don't know if it will be communicated within the interface that you can't cancel a swap.

**Comment by anxolin** [comment]
Yes I agree with @MareenG suggestion. Probably this is inherited from Uniswap. I think we should clear all not pending transaction. I wouldn't let the user clear the pending one. 

Adding it to our backlog, with not so much prio.

**Comment by MareenG** [comment]
Additional Note: When I am clearing my activities during a pending trade.  I also don't have a get any further info of my running order.  
No insight whether it went through or expired.

**Comment by elena-zh** [comment]
Still reproducible in the v.017.2 and on the https://barn.cowswap.exchange/
